content/300-guides/025-migrate/300-seed-database.mdx

```diff
@@ -342,6 +343,55 @@ psql file.sql
 
 </TabbedContent>
 
+### User-defined arguments
+
+> This feature is available from version 4.15.0 and later.
+
+`prisma db seed` allows you to define custom arguments in your seed file that you can pass to the `prisma db seed` command. For example, you could define your own arguments to seed different data for different environments or partially seeding data in some tables.
+
+Here is an example seed file that defines a custom argument to seed different data in different environments:
+
+```js file="seed.js"
+import { parseArgs } from 'node:util'
+
+const options = {
+  environment: { type: 'string' },
+}
+
+async function main() {
+  const {
+    values: { environment },
+  } = parseArgs({ options })
+
+  switch (environment) {
+    case 'development':
+      /** data for your development */
+      break
+    case 'test':
+      /** data for your test environment */
+      break
+    default:
+      break
+  }
+}
+
+main()
+  .then(async () => {
+    await prisma.$disconnect()
+  })
+  .catch(async (e) => {
+    console.error(e)
+    await prisma.$disconnect()
+    process.exit(1)
+  })
+```
+
+You can then provide the `environment` argument when using `prisma db seed` by adding an [option terminator](https://github.com/commandlineparser/commandline/wiki/CommandLine-Grammar#options-terminator-dashdash---) — `--` —, followed by your custom arguments:
```

Comment:
Note: I’ve found a reference to a standard, they call it a "delimiter"
https://unix.stackexchange.com/a/452793